docs/2.0.0-Beta/litmus-namespaced-2.0.0-Beta.yaml

```diff
@@ -168,6 +164,18 @@ spec:
               value: "litmuschaos/chaos-runner:1.13.3"
             - name: LITMUS_CHAOS_EXPORTER_IMAGE
               value: "litmuschaos/chaos-exporter:1.13.3"
+            - name: SERVER_SERVICE_NAME
```

Comment:
For the namespace, we don't need these details. User have to set the endpoint manually in the env- "PORTAL_ENDPOINT"

Reply:
In namespace mode why do user need to hardcode this PORTAL_ENDPOINT?

Reply:
Because namespace mode RBAC need cluster level permission

Reply:
I have tried this out in namespace mode and it worked fine also on a very restricted namespace mode of Okteto

Reply:
Have you tried to connect an agent using namespace mode? 

I think we are retrieving the node details which requires cluster level permission https://github.com/litmuschaos/litmus/blob/879b9cf303706bfffd7a366342eea7579dfb8a25/litmus-portal/graphql-server/pkg/k8s/cluster.go#L195

Reply:
Will test on all platforms and revert back.

Reply:
yes @rajdas98 i am testing it for all cases, will update the PR accordingly if needed.